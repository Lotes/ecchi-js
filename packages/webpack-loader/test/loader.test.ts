import { expect, test } from 'vitest';
import compiler from './compiler.js';
import { join } from 'path';

test('ecchi-loader', async () => {
  const stats = await compiler(join(__dirname,  '..',  '..',  '..',  'resources', 'Blog.ecchi'), { name: 'Alice' });
  const output = stats.toJson({ source: true })
  expect(output.modules![0].source).toMatchSnapshot();
});