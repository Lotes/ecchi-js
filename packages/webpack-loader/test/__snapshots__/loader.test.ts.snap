// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ecchi-loader 1`] = `
"/***********************************************
 * This file was generated by @ecchi-ts/webpack-loader v2
 * Is this in "webpack mode": true
 ***********************************************/

"
`;
